### PR TITLE
chore: Add `toolbar` example 

### PR DESCRIPTION
**Pull request by leonardoelias**
Adds a "toolbar" example https://github.com/reakit/reakit/issues/939.

## Toolbar
Screen Shot 2021-12-28 at 23 47 44

## Toolbar with menu
Screen Shot 2021-12-29 at 00 40 48
e-2f496bddc296.png">

### CODE REVIEW

**Comment by leonardoelias** [comment]
hey @diegohaz  is this how you were thinking?

File: packages/ariakit/src/toolbar/__examples__/toolbar/style.css [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    flex-row
+    w-fit
+    items-center
+    bg-canvas-4
+    dark:bg-canvas-4-dark
+  ;
+}
+
+.button {
+  @apply
```
Comment (leonardoelias) [inline packages/ariakit/src/toolbar/__examples__/toolbar/style.css:20]: is there a way to extend the button?

File: packages/ariakit/src/toolbar/__examples__/toolbar/style.css [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    text-canvas-2
+    hover:bg-canvas-4-hover
+    dark:hover:bg-canvas-4-dark-hover
+    dark:bg-canvas-4-dark
+    text-sm
+  ;
+}
+
+.button svg {
+  @apply sm:block hidden;
```
Comment (leonardoelias) [inline packages/ariakit/src/toolbar/__examples__/toolbar/style.css:31]: here i am removing the icon when the screen reaches the width of sm. This so as not to break the playground

File: packages/ariakit/src/toolbar/__examples__/toolbar-with-menu/index.tsx [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+      <ToolbarItem as={Button} className="button">
+        <ImItalic />
+        Italic
+      </ToolbarItem>
+      <ToolbarItem as={Button} className="button">
+        <ImUnderline />
+        Underline
+      </ToolbarItem>
+      <ToolbarSeparator className="toolbar-separator" />
+      {/* @ts-ignore */}
```
Comment (leonardoelias) [inline packages/ariakit/src/toolbar/__examples__/toolbar-with-menu/index.tsx:63]: sorry about that, but i couldn't resolve it.

File: packages/ariakit/src/toolbar/__examples__/toolbar-with-menu/index.tsx [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+  ImParagraphRight,
+  ImUnderline,
+} from "react-icons/im";
+import "./style.css";
+
+const MoreItems = forwardRef((props, ref) => {
+  const menu = useMenuState({ orientation: "horizontal" });
+  return (
+    <>
+      {/* @ts-ignore */}
```
Comment (leonardoelias) [inline packages/ariakit/src/toolbar/__examples__/toolbar-with-menu/index.tsx:25]: sorry about that, but i couldn't resolve it.

Comment (diegohaz) [reply packages/ariakit/src/toolbar/__examples__/toolbar-with-menu/index.tsx:25]: What's the error message? Just let the CI throw it so we can find a solution.

File: packages/ariakit/src/toolbar/__examples__/toolbar/style.css (hunk 2 above)
Comment (diegohaz) [reply packages/ariakit/src/toolbar/__examples__/toolbar/style.css:31]: That makes sense. We can also allow horizontal scrolling on the playground preview in such cases.

**Comment by diegohaz** [comment]
Let's work on a single example per PR. You can open another PR for the `toolbar-menu` example later.

File: packages/ariakit/src/toolbar/__examples__/toolbar/style.css (hunk 1 above)
Comment (diegohaz) [reply packages/ariakit/src/toolbar/__examples__/toolbar/style.css:20]: What do you mean by extending the button? If we have to override a lot of styles, it's okay not to import the base button example and add everything here.

**Pull request closed without merging**